js/ui/lookup.js

```diff
@@ -1027,6 +1048,13 @@ const Lookup = DropDownList.inherit({
         });
     },
 
+    _dispose: function() {
+        clearTimeout(this._closeOnTargetScrollTimer);
+        this._closeOnTargetScrollTimer = null;
```

Comment:
```suggestion
        clearTimeout(this._closeOnTargetScrollTimer);
        this._closeOnTargetScrollTimer = null;
```

Let's move it to `_clean`
Repaint -> clean + render
Dispose -> dispose-specific code + clean